test/utils-test.js

```diff
@@ -164,4 +167,66 @@ describe('Utils', function() {
     assert.strictEqual(validator.bool('shouldBeTrue'), true);
     assert.strictEqual(validator.bool('shouldBeFalse'), false);
   });
+
+  describe('Custom insepct function', () => {
```

Comment:
nit: typo

Reply:
huh did I make the exact same typo in the TITLE of the PR? oh boy I need more coffee